plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/DataSourceProviderRegistry.java

```diff
@@ -83,6 +84,9 @@ public synchronized static DataSourceProviderRegistry getInstance() {
     private final Map<String, DataSourceOriginProviderDescriptor> dataSourceOrigins = new LinkedHashMap<>();
     private final Map<String, DBPDriverSubstitutionDescriptor> driverSubstitutions = new HashMap<>();
 
+    private final List<Runnable> afterReloadListeners = new CopyOnWriteArrayList<>();
```

Comment:
Please reuse `DBPRegistryListener` instead of adding a whole new set of listeners